### PR TITLE
Switch to hosts next

### PR DESCRIPTION
**Pull request by fabriziosestito**
Switches to the new hosts endpoint based on the projector systems.
Hosts related tags api endpoints have been also refactored using the new hosts service.
The old consul service is kept since it is still used in the sap system lists (not yet strangled).

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]
LGTMd!

**Review by dottorblaster — APPROVED** [review]

File: web/app.go [hunk 1]
```diff
@@ -166,24 +166,23 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.StaticFS("/static", http.FS(assetsFS))
 	webEngine.GET("/", HomeHandler)
 	webEngine.GET("/about", NewAboutHandler(deps.subscriptionsService))
-	webEngine.GET("/hosts", NewHostListHandler(deps.consul, deps.tagsService))
-	webEngine.GET("/hosts-next", NewHostListNextHandler(deps.hostsNextService))
+	webEngine.GET("/hosts", NewHostListNextHandler(deps.hostsService))
```
Comment (arbulu89) [inline web/app.go:169]: Is intentional to keep the `Next` word here?

File: web/app.go [hunk 2]
```diff
@@ -100,8 +100,8 @@ func DefaultDependencies(config *Config) Dependencies {
 	araService := ara.NewAraService(viper.GetString("ara-addr"))
 	checksService := services.NewChecksService(araService, db)
 	subscriptionsService := services.NewSubscriptionsService(db)
-	hostsService := services.NewHostsService(consulClient)
-	hostsServiceNext := services.NewHostsNextService(db)
+	hostsService := services.NewHostsConsulService(consulClient)
```
Comment (arbulu89) [inline web/app.go:103]: This naming is confusing as `hostsService` is used as `hostsConsulService` in the future and `hostsServiceNext` as `hostsService`.

**Review by arbulu89 — COMMENTED** [review]
Coming late, but I have found some couple of things worth mentioning

File: web/app.go (hunk 1 above)
Comment (fabriziosestito) [reply web/app.go:169]: thanks it is a leftover im going to address it in a separate PR

File: web/app.go (hunk 2 above)
Comment (fabriziosestito) [reply web/app.go:103]: thanks like above i will clean it up in a separate PR

**Pull request merged**